test/jdk/javax/swing/JComboBox/6559152/bug6559152.java

```diff
@@ -77,19 +81,20 @@ private void setupUI() {
     }
 
     private void test() throws Exception {
-        robot = new ExtendedRobot();
-        robot.waitForIdle();
         testImpl();
         robot.waitForIdle();
         checkResult();
     }
 
     private void testImpl() throws Exception {
-        robot.type(KeyEvent.VK_DOWN);
+        robot.keyPress(KeyEvent.VK_DOWN);
```

Comment:
Extended robot calls "delay()" and "waitForIdle()" twice for each "type()", so it does not look like to roor cause, it is pretty similar to the updated code.

Reply:
Extended robot uses 20ms delay between events whereas we are using 100ms for setAutoDelay for other tests and we have seen timing does matter in mach5 testing, so although it's similar, time value differs.

Reply:
Not sure that such a difference is big enough, on macOS it is not possible to click faster than once per 50ms. I suggest looking to this comment in the JBS:
> 2. Test fails when run in group like running full JComboBox with jtreg. Looks like some execution problem. 
To confirm that you can press the "shift" or "cmd" or "alt" key during the test run and it will fail. So most probably some other test which was executed before caused the test to fail.